common/install_openblas.sh

```diff
@@ -14,6 +14,11 @@ DYNAMIC_ARCH=1
 TARGET=ARMV8
 CFLAGS=-O3
 "
+if [ "$DESIRED_CUDA" = "cu124" ]; then
```

Comment:
I think we need to use BASE_CUDA_VERSION 12.4 here instead 
Looks like DESIRED_CUDA is not set

Reply:
Thanks for checking, indeed it was not set. Let me use GPU_ARCH_TYPE: cuda-aarch64 instead